DICOMPlugins/DICOMSegmentationPlugin.py

```diff
@@ -462,7 +462,6 @@ def __init__(self, segmentationNode, contentCreatorName=None):
     self.contentCreatorName = contentCreatorName if contentCreatorName else "Slicer"
 
     self.tempDir = slicer.util.tempDirectory()
-    os.mkdir(self.tempDir)
```

Comment:
This is a bugfix from my previous PR - this will fail in the previous version since `slicer.util.tempDirectory()` already creates the directory. Apologies for not testing this before...